csharp/extractor/Semmle.Extraction.CSharp.DependencyFetching/DependencyManager.cs

```diff
@@ -237,14 +237,32 @@ private void AddNetFrameworkDlls(ISet<string> dllPaths, ISet<string> frameworkLo
             var packagesInPrioOrder = FrameworkPackageNames.NetFrameworks;
 
             var frameworkPath = packagesInPrioOrder
-                    .Select((s, index) => (Index: index, Path: GetPackageDirectory(s)))
-                    .FirstOrDefault(pair => pair.Path is not null);
+                .Select((s, index) => (Index: index, Path: GetPackageDirectory(s)))
+                .FirstOrDefault(pair => pair.Path is not null);
 
             if (frameworkPath.Path is not null)
             {
-                dllPaths.Add(frameworkPath.Path);
-                frameworkLocations.Add(frameworkPath.Path);
-                progressMonitor.LogInfo($"Found .NET Core/Framework DLLs in NuGet packages at {frameworkPath.Path}. Not adding installation directory.");
+                var versionFolders = new DirectoryInfo(frameworkPath.Path)
+                    .EnumerateDirectories("*", new EnumerationOptions { MatchCasing = MatchCasing.CaseInsensitive, RecurseSubdirectories = false })
+                    .OrderByDescending(d => d.Name) // TODO: Improve sorting to handle pre-release versions.
```

Comment:
Small comment: Sorting can be avoided to find the "correct" framework. This is not case, if it is important that the progressmonitor prints the version folder names in sorted order (I suppose this this the case).